src/renderer/components/+catalog/catalog-category-label.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import React from "react";
+
+interface CatalogCategoryLabelProps {
+  label: string | React.ReactNode;
+  badge?: React.ReactNode;
+}
+
+/**
+ * Display label for Catalog Category for the Catalog menu
+ */
+export const CatalogCategoryLabel = ({ label, badge }: CatalogCategoryLabelProps) =>
```

Comment:
I think that this should take only a `Category` as its input.

Reply:
This was discussed here before it was changed to take in just label and badge: https://github.com/lensapp/lens/pull/5156/files/799670ef314c3b9e1098bee564493267b9569b05#diff-8dc147d693864fb59061e8ac261091a3d88a6ae6f705895e62910a8307c61b51

Reply:
Interesting, I agree with the first part of that comment, but not really the second since this component is specifically about doing all that logic, I don't see the benefit of making the caller provide it.

Reply:
@jweak Thoughts?

Reply:
I think a representational component like Label should not do that logic.

Reply:
Interesting, may I garner some insight then. 

 would have said that a presentation component was the best place to do it because it encapsulates the behaviour. The user of this component doesn't care if it displays or not.

Reply:
I can see both implementations have merit. My point of would be to have a numbered components like `<Label>foo</Label>` and `<Header size={2}>catalog</Header>` and not create a new Label or Header component for each object we would like to display something.  

Anyway, if you feel strongly that it's better the other way, I'm fine with changing it back. It was just a suggestion.

Reply:
I don't feel terribly strong about it, I just wanted to know you thought pattern.

I agree 100% that there should be some base component for doing the some rendering but I also think that there should be a separate component for doing the logic (and not having it be done in line).

Reply:
Waiting for 1 more approval.

Reply:
I guess it's two against one for changing it back to taking the category so maybe change it and let's approve this and get it merged.

Reply:
It's changed back now